app/src/pages/LandingPage/Section2/CardsSection2.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+function Section2() {
+  return (
+    <>
+      <div className='h-[433px] grid grid-cols-2 w-full'>
+        <div className=" flex justify-center items-center border-black border-t-2 border-r-2 bg-[url('./Rectangles.svg')] bg-no-repeat bg-cover">
+          <p className='w-[330px] text-sm'>
+            DUN gives each project its own space. Easy and efficient!
+          </p>
+        </div>
+        <div className='bg-[#C5D4D2] flex justify-center items-center border-t-2 border-black'>
+          <img className='w-[499px] h-[342px]' src='./image 131.jpg' />
+        </div>
+      </div>
+      <div className='h-[433px] grid grid-cols-2 w-full'>
+        <div className='bg-[#CBB9CF] flex justify-center items-center border-r-2 border-t-2 border-black'>
+          <img className='w-[499px] h-[342px]' src='./image 132.jpg' />
+        </div>
+        <div className="flex justify-center items-center border-black bg-[url('./Squares.svg')] bg-no-repeat bg-cover border-t-2 ">
+          <p className='w-[330px] text-sm'>
+            One topic equals one neat spot for all files and info related to that case. <br />
+            <br />
+            P.S.Courtesy of DUN.
+          </p>
+        </div>
+      </div>
+      <div className='h-[433px] grid grid-cols-2 w-full'>
+        <div className="flex justify-center items-center border-t-2 border-r-2 border-black bg-[url('./Lines.svg')] bg-no-repeat bg-cover">
+          <p className='w-[330px] text-sm'>
+            DUN gives each project its own space. Easy and efficient!
+          </p>
+        </div>
+        <div className='bg-[#F5CBBC] flex justify-center items-center border-t-2 border-black'>
+          <img className='w-[499px] h-[342px]' src='./image 133.jpg' />
+        </div>
+      </div>
+      <div className='h-[433px] grid grid-cols-2 w-full'>
+        <div className='bg-[#FFF5D2] flex justify-center items-center border-r-2 border-t-2 border-black'>
+          <img className='w-[499px] h-[342px]' src='./image 134.jpg' />
+        </div>
+        <div className="flex justify-center items-center border-t-2 border-black bg-[url('./Plaid.svg')] bg-no-repeat bg-cover">
+          <p className='w-[330px] text-sm'>
+            Enjoy a clear and simple interface with DUN. No fuss, just straightforward ease!
+          </p>
+        </div>
+      </div>
```

Comment:
As you can see we have 4 similar entities - so I suggest to create component for that
```typescript
interface IFeatureCard {
    index: number;
    title: string;
    bg: string;
    image: React.ReactNode; 
}

const FEATURES: Omit<IFeatureCard, 'index'> = [{
        title: 'DUN gives each project its own space. Easy and efficient!',
        bg: '#C5D4D2',
        image: <img src='path-to-image' alt='description-image' />
    },
    ...
]

const FeatureCard = ({ title, color, image, index }: IFeatureCard) => {
    const isEven = index % 2 === 0
    
    return <div className={clsx('flex', isEven ? 'flex-row' : 'flex-row-reverse')}>
        <div className='text-lg'>{title}</div>
        <div className={clsx(bg ? `bg-${bg}` : 'bg-transparent')}>{image}</div>
    </div>
}


// use case
<div>
    {FEATURES.map((item, index) => <FeatureCard key={'feature-' + index} index={index} {...item} />)}
</div>
```